accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/AccessioningService.java

```diff
@@ -27,18 +27,39 @@
 
 import java.util.List;
 
+/**
+ * Service for retrieval and modifications of object accessions.
+ *
+ * @param <MODEL> Type of the objects identified by the accessions
+ * @param <HASH> Hash value of the fields that uniquely identify the object to be accessioned
```

Comment:
it is the Type of Hash

Reply:
Could you please put the comment at https://github.com/EBIvariation/accession-commons/pull/55#discussion_r263829553 so the it will ease discussion

Reply:
@selvaebi is right, the template specifies types, not values. New suggestion:

```suggestion
 * @param <HASH> Type of the hash calculated based on the fields that uniquely identify an accessioned object
```